ui-tests/src/test/java/org/openmrs/reference/page/RegistrationSummaryPage.java

```diff
@@ -0,0 +1,26 @@
+package org.openmrs.reference.page;
+
+import org.openmrs.uitestframework.page.Page;
+import org.openqa.selenium.By;
+
+public class RegistrationSummaryPage extends Page {
+	
+    private static final By RELATIONSHIP = By.id("relationships-info-edit-link");
+    private static final By DEMOGRAPHICS = By.id("demographics-edit-link");
+    private  static final By CONTACT_INFO = By.id("contactInfo-edit-link");
+
+    public RegistrationSummaryPage(Page parent) {
+        super(parent);
+    }
+
+    public EditPatientRelationshipPage goToEditPatientRelationship(){
+        clickOnLast(RELATIONSHIP);
+        return new EditPatientRelationshipPage(this);
+    }
+
+    @Override
+    public String getPageUrl() {
+        return "/registrationapp/registrationSummary.page";
+    }
+}
+
```

Comment:
Minor comment on code line `26 `can be translated at my review [above ](https://github.com/openmrs/openmrs-distro-referenceapplication/pull/521/files#r694666673)

Reply:
I  think everything is clean here according to java conventions

Reply:
Refer to the  caption [above](https://github.com/openmrs/openmrs-distro-referenceapplication/pull/521#discussion_r694761380) !

Reply:
It would be more clear and clean to leave this kind of commit.

Reply:
> It would be more clear and clean to leave this kind of commit.

If you do not mind you may want to check my comment [here](https://github.com/openmrs/openmrs-distro-referenceapplication/pull/521#discussion_r694884486) !